afdko/.SRCINFO

```diff
@@ -1,21 +1,28 @@
 pkgbase = afdko
 	pkgdesc = Adobe Font Development Kit for OpenType
 	pkgver = 4.0.1
-	pkgrel = 1
+	pkgrel = 6
 	url = https://github.com/adobe-type-tools/afdko
 	arch = x86_64
-	license = custom
+	license = Apache-2.0
 	checkdepends = python-pytest
+	checkdepends = python-virtualenv
```

Comment:
Nope, this is going to be wrong. We should never need virtualenv to build a package.